test/lib-publish.spec.js

```diff
@@ -4,14 +4,28 @@ describe('skyux-deploy lib publish', () => {
 
   const mock = require('mock-require');
 
+  const loggerInfoSuccess = 'Successfully published.';
+
+  let loggerMock;
   let publishSpaMock;
   let publishStaticMock;
   let publish;
 
+  async function callPublish(isStaticClient) {
+    return publish({
+      azureStorageAccessKey: 'abc',
+      isStaticClient: isStaticClient,
+      name: 'custom-name',
+      version: 'custom-version'
+    });
+  }
+
   beforeEach(() => {
+    loggerMock = jasmine.createSpyObj('logger', ['info', 'error', 'bobby']);
```

Comment:
What's `bobby` used for?

Reply:
To maintain my presence in this repo.  J/K, I was debugging.  I'll remove it.